src/core/utils/request/onUnhandledRequest.ts

```diff
@@ -24,7 +24,9 @@ export async function onUnhandledRequest(
   const url = new URL(request.url)
   const publicUrl = toPublicUrl(url) + url.search
 
-  const unhandledRequestMessage = `intercepted a request without a matching request handler:\n\n  \u2022 ${request.method} ${publicUrl}\n\nIf you still wish to intercept this unhandled request, please create a request handler for it.\nRead more: https://mswjs.io/docs/getting-started/mocks`
+  const requestBody = await request.text()
```

Comment:
We should clone the request before reading it.

```suggestion
  const requestBody = await request.clone().text()
```